src/app/projects/components/create-new-attribute/create-new-attribute.component.ts

```diff
@@ -324,6 +325,9 @@ export class CreateNewAttributeComponent implements OnInit, OnDestroy {
       this.updatedThroughWebsocket = true;
       this.currentAttributeQuery$.refetch();
     } else if (msgParts[1] == 'calculate_attribute') {
+      if (msgParts[2] == 'progress') {
+        this.currentAttribute.progress = Number(msgParts[4]);
+      }
       this.attributesQuery$.refetch();
       this.currentAttributeQuery$.refetch();
```

Comment:
i think we dont need both. If the progress is updated directly we don't need to refetch the whole thing. (at least I think so). If we do need to refetch then the progress update itself is unnecessary.